packages/react/src/components/Provider/Provider.context.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+export type ColorMode = 'light' | 'dark';
+
+export interface ProviderContext {
```

Comment:
is this a generic provider?

Reply:
oh this is future work. let me delete it